### PR TITLE
Making eventlog scrollable

### PR DESCRIPTION
**Pull request by pradeepvairamani**
Fix for issue #7

### CODE REVIEW

File: coup-client/src/components/game/CoupStyles.css [hunk 1]
```diff
@@ -357,6 +357,8 @@
         position: absolute;
         right: 10VW;
         top: 10VH;
+        overflow-y: scroll;
```
Comment (Cheneth) [inline coup-client/src/components/game/CoupStyles.css:360]: This needs to be added to the mobile section as well, but I'll just merge it and add it myself.

Comment (Cheneth) [reply coup-client/src/components/game/CoupStyles.css:360]: Actually, we can just add it to the above section only, to be more concise

Comment (pradeepvairamani) [reply coup-client/src/components/game/CoupStyles.css]: Pushed some changes. It seems to work fine in both mobile and web

image

image

**Comment by Cheneth** [comment]
I'm also going to edit the CSS a bit to make sure the "Event Log" title doesn't scroll with it

**Pull request merged**